providers/aws/aws_service.go

```diff
@@ -31,7 +31,7 @@ func (s *AWSService) generateSession() *session.Session {
 	sess := session.Must(session.NewSessionWithOptions(session.Options{
 		Profile:                 s.GetArgs()["profile"].(string),
 		AssumeRoleTokenProvider: stscreds.StdinTokenProvider,
-		SharedConfigState:       session.SharedConfigEnable,
+		SharedConfigState:       session.SharedConfigStateFromEnv,
```

Comment:
This change prompted the need for some documentation: https://github.com/GoogleCloudPlatform/terraformer/pull/199